src/pages/resultsView/enrichments/ExpressionEnrichmentsTable.tsx

```diff
@@ -147,7 +147,7 @@ export default class ExpressionEnrichmentTable extends React.Component<IExpressi
         columns[ExpressionEnrichmentTableColumnType.P_VALUE] = {
             name: "p-Value",
             render: (d: ExpressionEnrichmentRow) => <span style={{whiteSpace: 'nowrap'}}>{toConditionalPrecision(d.pValue, 3, 0.01)}</span>,
-            tooltip: <span>Derived from Fisher Exact Test</span>,
+            tooltip: <span>Derived from Studen's t-test</span>,
```

Comment:
@ersinciftci Student's

Reply:
@inodb lol, I copy pasted from @jjgao 's comment in https://github.com/cBioPortal/cbioportal/issues/4435 so that I wouldn't make a typo 😄 good catch